testing/kiconthemes/APKBUILD

```diff
@@ -7,11 +7,13 @@ pkgdesc="Support for icon themes"
 arch="all"
 url="https://community.kde.org/Frameworks"
 license="LGPL-2.1-only"
-depends_dev="qt5-qtsvg-dev karchive-dev ki18n-dev kcoreaddons-dev kconfigwidgets-dev kwidgetsaddons-dev kitemviews-dev"
+depends_dev="qt5-qtsvg-dev karchive-dev ki18n-dev kcoreaddons-dev kconfigwidgets-dev
+	kwidgetsaddons-dev kitemviews-dev"
```

Comment:
I personally prefer all dependencies on a single line, my editor will break the lines internally.